### ISSUE TITLE
[stable/elastic-stack] Target Namespace

### ISSUE THREAD

**Issue by navalta3030** [helm/charts#22011]
**Is your feature request related to a problem? Please describe.**
Elastic-stack chart is being deployed on "monitoring-elk" namespace, however main application is located in default namespace. Logstash doesn't obtain the logs of default. 

**Describe the solution you'd like**
Be able to pass a parameter on helm that would specify the target namespace.

**Describe alternatives you've considered**
Use default namespace to deploy the elastic. But that would be unorganized in production.

**Additional context**
Basically trying to find a way to pass in a target namespace.
